### PR TITLE
dependencies update

### PR DESCRIPTION
**Pull request by Xanclry**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,8 +19,8 @@ targetCompatibility = JavaVersion.VERSION_11
 
 ext {
     grpcVersion         = '1.32.1'
-    protobufVersion     = '3.19.4'
-    kotlin_version      = '1.3.72'
+    protobufVersion     = '3.21.12'
```
[the remark this answers is not in the document]
Comment (Xanclry) [reply build.gradle:22]: I haven't found any actual grpc/protobuf compatibility notice, but protobuf's docs says: "Protobuf minor version releases are backwards-compatible. If your code can build/run against the old version, it's expected to build/run against the new version as well."

[the remark this answers is not in the document]
Comment (OptimumOpium) [reply build.gradle:22]: Okay, we shouldn't have any problems then. But just to be sure, could you please try to update protobuf version in a repo that has services (like grpc-check1 for example)? Let's check if it compiles corrrectly

File: build.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@ sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
 ext {
-    grpcVersion         = '1.32.1'
+    grpcVersion         = '1.48.1'
```
Comment (OptimumOpium) [inline build.gradle:21]: Just noticed that it is not used anywhere. Maybe we should remove it? The same goes for other properties that are not used

Comment (Xanclry) [reply build.gradle:21]: it used here: artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"

Comment (Xanclry) [reply build.gradle:21]: but I removed slfj4_version and kotlin_version

File: build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (Xanclry) [reply build.gradle:22]: grpc-check1 compiles with protobuf v3.21.12 and bom v4.2.0

File: build.gradle (hunk 2 above)
Comment (OptimumOpium) [reply build.gradle:21]: > it used here: artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}"

Sorry, missed that.

File: build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (OptimumOpium) [reply build.gradle:22]: Great

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**